### PR TITLE
DDS CMake improvement

### PR DESCRIPTION
**Pull request by jjomier**
Modify CMake processing to avoid finding package holoscan when not using DDS

### CODE REVIEW

**Comment by tbirdso** [comment]
@jjomier I see a build error when attempting to build DDS with this branch:

```
fatal error: ../../../operator_util.hpp: No such file or directory
```

It looks like this PR goes the extra mile to align DDS to HoloHub infrastructure, which I agree we should pursue, but does more work than required to address the issue at hand. I'm going to open a companion PR with the minimum fix and then I will help track down the organization issue here.

**Comment by jjomier** [comment]
> @jjomier I see a build error when attempting to build DDS with this branch:
> 
> ```
> fatal error: ../../../operator_util.hpp: No such file or directory
> ```
> 
> It looks like this PR goes the extra mile to align DDS to HoloHub infrastructure, which I agree we should pursue, but does more work than required to address the issue at hand. I'm going to open a companion PR with the minimum fix and then I will help track down the organization issue here.

We need to target include the base operator directory - which was removed. I'm pushing a fix

**Comment by jjomier** [comment]
> > @jjomier I see a build error when attempting to build DDS with this branch:
> > ```
> > fatal error: ../../../operator_util.hpp: No such file or directory
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > It looks like this PR goes the extra mile to align DDS to HoloHub infrastructure, which I agree we should pursue, but does more work than required to address the issue at hand. I'm going to open a companion PR with the minimum fix and then I will help track down the organization issue here.
> 
> We need to target include the base operator directory - which was removed. I'm pushing a fix

Actually just need to point to the right path.

**Comment by tbirdso** [comment]
```
fatal error: dds/shapes/subscriber.hpp: No such file or directory
```

File: operators/dds/video/dds_video_publisher/python/publisher_pydoc.hpp
Comment (jjomier) [inline operators/dds/video/dds_video_publisher/python/publisher_pydoc.hpp:1]: Should be dds_video_publisher

Comment (tbirdso) [reply operators/dds/video/dds_video_publisher/python/publisher_pydoc.hpp:1]: Ah, I see the "#include" statement I referenced was in fact the issue with respect to naming conventions. Fixed.

**Comment by jjomier** [comment]
LGTM! Thanks for fixing @tbirdso

**Comment by ibstewart** [comment]
It looks good to me, too, and I've tested it on my end without any issues. Thanks!

**Pull request merged**